src/interfaces/author.ts

```diff
@@ -0,0 +1,9 @@
+type Image = {
+    url: string;
+};
+
+export interface Author {
+    name: string;
+    bio: string;
+    image: Image;
+}
```

Comment:
Won't we need a `url` or `slug` field for the authors in order to link to their corresponding author page?

Reply:
Good catch.will add.